tests/examples/action_result_PASS.json

```diff
@@ -11,35 +11,33 @@
             "@id": "kb:action-1",
             "@type": "action:Action",
             "rdfs:comment": "This node is some action that has some ObservableObjects as results.  By the ontology, the results need to be some UcoObject or subclass of UcoObject.  They are serialized here as ObservableObjects, and are redundantly assigned types of some of their superclasses.  For completeness-tracking, let the id slug's number be a binary number tracking which superclasses are present, 2^0=core:UcoObject, 2^1=core:Item, 2^2=observable:Observable.",
-            "core:hasFacet": {
-                "@type": "action:ActionReferencesFacet",
-                "action:result": [
-                    {
-                        "@id": "kb:node-0"
-                    },
-                    {
-                        "@id": "kb:node-1"
-                    },
-                    {
-                        "@id": "kb:node-2"
-                    },
-                    {
-                        "@id": "kb:node-3"
-                    },
-                    {
-                        "@id": "kb:node-4"
-                    },
-                    {
-                        "@id": "kb:node-5"
-                    },
-                    {
-                        "@id": "kb:node-6"
-                    },
-                    {
-                        "@id": "kb:node-7"
-                    }
-                ]
-            }
+            "core:name": "Open File",
```

Comment:
There was not a need from the CP to add this line.  Would you mind force-pushing this commit to revise it to all but adding this line?

Reply:
This should now be resolved.

Reply:
Confirmed.